.github/workflows/build-podman.yaml

```diff
@@ -290,3 +290,73 @@ jobs:
         push: ${{ github.event_name != 'pull_request' }}
         labels: ${{ steps.metaContributors.outputs.labels }}
         platforms: linux/amd64
+
+  BuildPython:
+    needs: BuildBase
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Docker meta for KubeSphere
+        id: meta
+        if: github.repository_owner == 'kubesphere'
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            kubespheredev/builder-python
+            ghcr.io/${{ github.repository_owner }}/builder-python
+          tags: |
+            type=schedule
+            type=ref,event=branch,suffix=-podman
+            type=ref,event=pr,suffix=-podman
+            type=semver,pattern=v{{version}}-podman
+            type=sha,suffix=-podman
+      - name: Docker meta for Contributors
+        id: metaContributors
+        if: github.repository_owner != 'kubesphere'
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            ghcr.io/${{ github.repository_owner }}/builder-python
+          tags: |
+            type=schedule
+            type=ref,event=branch,suffix=-podman
+            type=ref,event=pr,suffix=-podman
+            type=semver,pattern=v{{version}}-podman
+            type=sha,suffix=-podman
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USER }}
+          password: ${{ secrets.DOCKER_HUB_SECRETS }}
+      - name: Login to GHCR
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GHCR_TOKEN }}
+      - name: Build and push Docker images
+        uses: docker/build-push-action@v2.4.0
+        if: github.repository_owner == 'kubesphere'
+        with:
+          context: python
+          file: python/podman/Dockerfile
+          tags: ${{ steps.meta.outputs.tags }}
+          push: ${{ github.event_name != 'pull_request' }}
+          labels: ${{ steps.meta.outputs.labels }}
+          platforms: linux/amd64
+      - name: Build and push Docker images for Contributors
+        uses: docker/build-push-action@v2.4.0
+        if: github.repository_owner != 'kubesphere'
+        with:
+          context: python
+          file: python/podman/Dockerfile
+          tags: ${{ steps.metaContributors.outputs.tags }}
+          push: ${{ github.event_name != 'pull_request' }}
+          labels: ${{ steps.metaContributors.outputs.labels }}
+          platforms: linux/amd64
```

Comment:
Please keep an empty line at the end of each file. See also [here](https://www.google.com.hk/search?q=why+add+a+new+line+at+the+end+of+this+file&oq=&sourceid=chrome&ie=UTF-8).

Reply:
Done. 
BTW, We can use [EditorConfig](https://editorconfig.org/) to guarantee that behavior.

Reply:
Good idea.